### PR TITLE
feat: [CDS-83191]: Sample pipeline fixed for api docs

### PR DESCRIPTION
**Pull request by Aninditaa-ch**
## Describe your changes
Changes made to the api docs to Execute a Pipeline with Runtime Input YAML with accordance to the sample pipeline created in the docs:
https://apidocs.harness.io/tag/Pipeline-Execute/#operation/postPipelineExecuteWithInputSetYaml. 
Screenshot 2023-11-02 at 12 58 33 PM

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/54849)

### CODE REVIEW

**Comment by Aninditaa-ch** [comment]
trigger uts

**Comment by ankittiwari-harness** [comment]
can you add details on what we are tryingto do here. Screenshots

**Comment by Aninditaa-ch** [comment]
trigger uts

**Comment by Aninditaa-ch** [comment]
trigger uts

**Comment by Aninditaa-ch** [comment]
trigger messagecheck

**Comment by Aninditaa-ch** [comment]
trigger uts

**Review by shalini-agr — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/54849)* status: 1 file(s) to review. 1 change requests. 0 of 2 approvals obtained. 

___
*[`pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java` line 179 at r1](https://reviewable.io/reviews/harness/harness-core/54849#-NiEh1mF7PS1yWwRD1xX:-NiEh1mF7PS1yWwRD1xY:b-7t9azh) ([raw file](https://github.com/harness/harness-core/blob/dd1a0120c900439c5c156b0007c8a8757d69d7bf/pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java#L179)):*
> ```Java
>       + "                            approvers:\n"
>       + "                              userGroups: \"<+input>\"\n"
>       + "                                    - account._account_all_users:\n"
> ```

the indentation seems off, please verify once
___
*[`pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java` line 180 at r1](https://reviewable.io/reviews/harness/harness-core/54849#-NiEh7gG1WVrBWtl2UO7:-NiEh7gH7wcu1SLbf2dc:bpgwxg1) ([raw file](https://github.com/harness/harness-core/blob/dd1a0120c900439c5c156b0007c8a8757d69d7bf/pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java#L180)):*
> ```Java
>       + "                              userGroups: \"<+input>\"\n"
>       + "                                    - account._account_all_users:\n"
>       + "                                       - step:\n"
> ```

here as well

**Comment by prasadsatamharness** [comment]
@Aninditaa-ch - Looks good to me.

**Review by Aninditaa-ch — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/54849)* status: 1 file(s) to review. 1 change requests. 2. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java` line 179 at r1](https://reviewable.io/reviews/harness/harness-core/54849#-NiEh1mF7PS1yWwRD1xX:-NiIWzoQDOPKgTUO9s_F:b-6gayyd) ([raw file](https://github.com/harness/harness-core/blob/dd1a0120c900439c5c156b0007c8a8757d69d7bf/pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java#L179)):*
<details><summary><i>Previously, shalini-agr (Shalini Agrawal) wrote…</i></summary><blockquote>

the indentation seems off, please verify once
</blockquote></details>

I've changed it according to previous lines, can you see once now
___
*[`pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java` line 180 at r1](https://reviewable.io/reviews/harness/harness-core/54849#-NiEh7gG1WVrBWtl2UO7:-NiIX4V75f4mBZG9D4ek:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/dd1a0120c900439c5c156b0007c8a8757d69d7bf/pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java#L180)):*
<details><summary><i>Previously, shalini-agr (Shalini Agrawal) wrote…</i></summary><blockquote>

here as well
</blockquote></details>

done

**Review by SudarshanPurohit — DISMISSED** [review]

File: pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java [hunk 1]
```diff
@@ -176,25 +176,26 @@ public class PipelineAPIConstants {
       + "                          spec:\n"
       + "                            approvers:\n"
       + "                              userGroups: \"<+input>\"\n"
```
Comment (shalini-agr) [inline pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java:178]: remove \"<+input>\"\n" from here 
as we are providing the list value

**Review by shalini-agr — CHANGES_REQUESTED** [review]

File: pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java (hunk 1 above)
Comment (Aninditaa-ch) [reply pipeline-service/contracts/src/main/java/io/harness/apiexamples/PipelineAPIConstants.java]: done

**Review by shalini-agr — APPROVED** [review]

**Review by ankittiwari-harness — APPROVED** [review]

**Comment by Aninditaa-ch** [comment]
trigger uts

**Comment by Aninditaa-ch** [comment]
trigger reviewCheck

**Pull request merged**